include/zephyr/mgmt/hawkbit.h

```diff
@@ -154,7 +157,7 @@ struct hawkbit_runtime_config hawkbit_get_config(void);
 static inline int hawkbit_set_server_addr(char *addr_str)
 {
 	struct hawkbit_runtime_config set_config = {
-		.server_addr = addr_str, .server_port = 0, .auth_token = NULL};
+		.server_addr = addr_str, .server_port = 0, .auth_token = NULL, .tls_tag = 0};
```

Comment:
Hmm.. imagine that if we do not provide these functions, we probably don't need to modify this file at all

Reply:
Well now we have them. and if it's getting to long we could still change it to 
```
struct hawkbit_runtime_config set_config = {0};
set_config.server_addr = addr_str;
```